### ISSUE TITLE
V3 summoner.current_match.duration issues

### ISSUE THREAD

**Issue by SuperFrosty** [meraki-analytics/cassiopeia#124]
Code used
```python
from cassiopeia.core import Summoner

summoner = Summoner(name="Riviere", region="OCE")
current_match = summoner.current_match
print(current_match.duration)
```
On Windows specifically, an OSError 22 is called:
Traceback
```bash
Traceback (most recent call last):
  File "C:\Anaconda3\lib\site-packages\merakicommons\ghost.py", line 41, in wrapper
    return method(*args, **kwargs)
  File "C:\Anaconda3\lib\site-packages\cassiopeia\core\spectator.py", line 362, in duration
    return datetime.datetime.fromtimestamp(self._data[CurrentGameInfoData].duration / 1000)
  File "C:\Anaconda3\lib\site-packages\cassiopeia\core\spectator.py", line 182, in duration
    return self._dto["gameLength"]
KeyError: 'gameLength'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Anaconda3\lib\site-packages\merakicommons\ghost.py", line 87, in __get__
    return self.fget(obj)
  File "C:\Anaconda3\lib\site-packages\merakicommons\ghost.py", line 43, in wrapper
    raise GhostLoadingRequiredError(str(error))
merakicommons.ghost.GhostLoadingRequiredError: 'gameLength'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\redacted\OneDrive\Bots\ClanBot\test.py", line 6, in <module>
    print(current_match.duration)
  File "C:\Anaconda3\lib\site-packages\merakicommons\ghost.py", line 93, in __get__
    return self.fget(obj)
  File "C:\Anaconda3\lib\site-packages\merakicommons\ghost.py", line 41, in wrapper
    return method(*args, **kwargs)
  File "C:\Anaconda3\lib\site-packages\cassiopeia\core\spectator.py", line 362, in duration
    return datetime.datetime.fromtimestamp(self._data[CurrentGameInfoData].duration / 1000)
OSError: [Errno 22] Invalid argument
```
On Linux (Ubuntu) this works fine, however there is another issue where it always returns: 
`1970-01-01 00:00:00`

**Comment by jjmaldonis** [comment]
Reading a bit on google, it seems that the issue on windows and linux arise from the same problem -- that the timestamp being passed in isn't valid. Maybe we shouldn't be dividing by 1000 here, or maybe it is more subtle than that.

If you have a chance to look into it that would be great!

**Comment by jjmaldonis** [comment]
Should have been closed by 920634f97686bf929f8a5892c37447833d1b0290
